nightwatch/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENV=$1
+[ -z "$ENV" ] && ENV='default'
+
+if [ "$ENV" != "default" ] ; then
+    CRED="../mycredentials"
+    [ ! -f $CRED ] && echo "Could not find credentials file: $CRED" && exit 1
+    CRED="env `cat $CRED`"
```

Comment:
`env` shouldn't run here (just `cat`), it should run when `nightwatch` is called on line 12